runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -373,6 +373,14 @@ public static UUID getStreamID(String string) {
         return UUID.nameUUIDFromBytes(string.getBytes());
     }
 
+    public static UUID getCheckpointStreamIdFromId(UUID streamId) {
+        return getStreamID(streamId.toString() + "_cp");
```

Comment:
Probably want to make the string lateral a static definition in streamview

Reply:
sound right.